### ISSUE TITLE
Problem about `useModel` API signature

### ISSUE THREAD

**Issue by hackape** [modern-js-dev/reduck#28]
https://github.com/modern-js-dev/reduck/blob/6381efec31d0307cc6d63e9a4ca52a9ea20f4c56/packages/store/src/model/useModel.ts#L49-L58

I feel this implementation is problematic. 

Say I want to supply an `actionSelector` but stick with the default state merging `stateSelector`, I'd write:

```js
const [fooAndBar, actions] = useModel(fooModel, barModel, null, myActionSelector)
```

Current implementation just doesn't allow such usage. 😑

**Comment by anc95** [comment]
That's a good question, and in the scenario you cite it is indeed problematic, I need to take the time to see how to optimize it a bit and not bring about breaking changes

**Comment by xuchaobei** [comment]
Fixed in #42
